cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -40,7 +40,7 @@
           -----------------------
           .m-global-eyebrow.m-global-eyebrow__list
             .wrapper
-              .m-global-eyebrow_tagline
+              p.a-tagline
```

Comment:
Extremely trivial, but the markup for this is using `div` still, not `p`. (I think `div` is the correct choice.)

Reply:
@jimmynotjim suggested `p`, though before I reworked this. Which would you'll like?

Reply:
I think `div`, as it's not even a complete sentence.

Reply:
Yeah, `div` seems fine and matched what we're doing with notifications. I only suggested `p` because it's read as one unit, even with inline elements breaking up the content within.

Reply:
Done!